openmetadata-service/src/main/java/org/openmetadata/service/migration/utils/v140/MigrationUtil.java

```diff
@@ -35,6 +38,32 @@ private MigrationUtil() {
     /* Cannot create object  util class*/
   }
 
+  public static void migrateGenericToWebhook(CollectionDAO collectionDAO) {
+    try {
+      List<String> jsonEventSubscription =
+          collectionDAO.eventSubscriptionDAO().listAllEventsSubscriptions();
+      for (String eventSubscription : jsonEventSubscription) {
```

Comment:
from what I remember postgres returns a `PGObject` and not a `String`.

Reply:
@TeddyCr that is needed only when working with handle and writing custom queries , when you do row.get()